### ISSUE TITLE
 How to get Loss landscape

### ISSUE THREAD

**Issue by ShenZheng2000** [ma-xu/pointMLP-pytorch#42]
Hello authors! I really appreciate your excellent work. For Fig. 4 (the Loss landscape) in your paper, I want to know the following information:
(1) The plotting code script, package, or repository
(2) The testing dataset 
(3) The model version 
Thanks!
Shen Zheng

**Comment by ma-xu** [comment]
Hi @ShenZheng2000 Thanks for your interest.
1. We firstly get the raw data by [lossland.py](https://drive.google.com/file/d/1bZL_ZRcfaJBWwrxnbajATBTCW6QhxsAI/view?usp=sharing), then we plot the figure by [plot.py] (https://drive.google.com/file/d/1_ygidzgere1r-wLthb9O2Rffp_FKba95/view?usp=sharing). Note that both scripts are not cleaned andn you may need to change some settiings accordingly (e.g., checkpoint paths, ranges, etc.). More details can be found here: Li, Hao, et al. "Visualizing the loss landscape of neural nets." NeurIPS 2018.
2. The testing dataset is ModelNet40, as indicated in Sec. Loss landscape.
3. We test PointMLP and PointMLP without skip connections.

Please let me know if you have any further questions.

**Comment by ShenZheng2000** [comment]
Thanks for your timely reply. Got it now.
